environment.yml

```diff
@@ -1,60 +1,38 @@
 name: geo_deep_env
 channels:
-  - anaconda
-  - conda-forge
   - pytorch
-  - defaults
+  - conda-forge
 dependencies:
-  - fiona>=1.8.0
-  - gdal>=3.0.0
   - boto3>=1.19.2
-  - cudatoolkit=10.2.89
+  - cudatoolkit>=11.3.0
+  - pytorch=1.10.1=py3.9_cuda11.3_cudnn8.2.0_0  # Linux
```

Comment:
Why not pytorch-1.10.2-py3.9_cuda11.3_cudnn8.2.0_0.  I think you have a typo a=b=c ?

Reply:
Just a sheep approach: 1.10.1 has 27k download from conda-forge, where as 1.10.2 has around 8k. We can switch to 1.10.2, I don't see any problem.

Reply:
The newer release gets fewer downloads. Makes sense :-). I think we should stick with the latest stable version. And stick to it for a while unless some major vulnerability comes out.

Reply:
Changes made.